### ISSUE TITLE
update.php reports that site is Drupal 7

### ISSUE THREAD

**Issue by Graham-72** [backdrop/backdrop-issues#1490]
Today I updated core to V 1.2.3 and was shown this
![requirements problem - deben rowing club 2016-01-08 06-22-25](https://cloud.githubusercontent.com/assets/10559274/12191950/0886a78c-b5d2-11e5-9e54-7da88c98bdc9.png)

also perhaps we could revise the status report about securing directores and have a backdrop link?
![status report - deben rowing club 2016-01-08 06-27-34](https://cloud.githubusercontent.com/assets/10559274/12191967/49b822d0-b5d2-11e5-9380-f8156b5916c9.png)

**Comment by Graham-72** [comment]
Looking at the code in update.inc I see that for some reason it has failed to recognise that this is already a Backdrop site.

**Comment by BWPanda** [comment]
@Graham-72 Is this still an issue for you, or has it been fixed since?

**Comment by klonos** [comment]
Yeah, this `You are running a current version of Drupal 7.` string is in https://github.com/backdrop/backdrop/blob/1.x/core/includes/update.inc#L130, and it's there for a reason.

I will go ahead and close this issue as outdated, as we had no other reports of this in years 😉

**Comment by indigoxela** [comment]
> I was testing some upgrades

So this happens *before* the upgrade actually starts? Because in that case the message seems correct - it's no Backdrop yet.
